modules/Core/module.php

```diff
@@ -1618,6 +1608,30 @@ public static function addDataToDashboardGraph($title, $data): void {
         }
     }
 
+    /**
+     * Fill missing days in an array containing graph data
+     *
+     * @param array $data Initial data to display on graph
+     * @param string $key Key to output data under
+     * @return array Formatted data under $key
+     */
+    public static function fillMissingGraphDays(array $data, string $key): array {
```

Comment:
Not a huge deal, but I think this would better fit in the main `Collection` class, rather than requiring other modules to use the `Core_Module` class

Reply:
I was debating this but in the end decided against moving it out - currently we have other graph related code in here which could also do with moving out, which was a bit out of scope for this performance improvements piece.